### PR TITLE
fix deprecations on julia v0.5+

### PR DESCRIPTION
**Pull request by tlnagy**
Currently NRRD is broken on julia 0.5+, this PR brings it up to speed.

### CODE REVIEW

**Comment by timholy** [comment]
Great, many thanks!

File: .travis.yml [hunk 1]
```diff
@@ -3,8 +3,9 @@ os:
   - linux
   - osx
 julia:
-  - 0.3
```
Comment (tkelman) [inline .travis.yml:6]: should update REQUIRE when you do this

File: src/NRRD.jl [hunk 2]
```diff
@@ -3,6 +3,7 @@ module NRRD
 using Images, SIUnits, SIUnits.ShortUnits, Compat, FileIO
 import Zlib
 import FixedPointNumbers
+import Compat: ASCIIString, UTF8String
```
Comment (tkelman) [inline src/NRRD.jl:6]: needs a minimum version dependency on Compat

**Comment by tlnagy** [comment]
Good catches @tkelman. I'll make a new PR.

**Pull request merged**